simple/rdm_shared_av.c

```diff
@@ -156,6 +156,7 @@ int main(int argc, char **argv)
 
 	opts = INIT_OPTS;
 	opts.options |= FT_OPT_SIZE;
+	opts.av_name = "foo";
```

Comment:
> Since server and client can run on the same system, av_name should be different.

That makes perfect sense, the error message wasn't super useful.